cernopendata/modules/fixtures/data/records/cms-tools-heavyion-examples.json

```diff
@@ -0,0 +1,116 @@
+    },
+    "use_with": {
+      "description": "Use this with CMS heavy-ion 2010 primary datasets",
+      "links": [
+        {
+          "recid": "14010"
+        },
+        {
+          "recid": "14011"
+        }
+      ] 
+    }
+  },
+  {
+    "abstract": {
+      "description": " <p><p>This is an example of the C++ code (and its configuration in Python) needed to extract information from CMS heavy-ion data and write them to a derived dataset. An example analysis routine is provided to illustrate how to analyse data in this derived output file. These instructions are valid to work with 2011 data. Snippets of this software can be used for physics analysis.</p> <p>The examples, within the code, have in-line comments to make it more understandable and to point potential users to the original information sources. It has been put together compiling the most important pieces of already-available <a href=\"https://github.com/cms-sw/cmssw\">CMSSW software</a>.</p></p> "
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Bylinkin, Alexandr"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_created": [
+      "2011"
+    ],
+    "date_published": "2020",
+    "experiment": "CMS",
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "467",
+    "run_period": [
+      "HIRun2011"
+    ],
+    "source_code_repository": {
+      "url": "https://github.com/cms-legacydata-analyses/HiForestProducerTool/tree/HIDiMuon2011"
+    },
+    "system_details": {
+        "description": "Use this code with the CMS Open Data VM environment or the corresponding Docker container",
+        "recid": "251",
+        "release": "CMSSW_4_4_7"
+    },
+    "title": "Usage example for CMS 2011 heavy-ion data",
+    "type": {
+      "primary": "Software",
+      "secondary": [
+        "Tool",
+        "Analysis",
+        "Workflow"
+      ]
+    },
+    "usage": {
+      "description": "<p>If you do not have the CERN Virtual Machine for 2011 CMS data installed, follow the instructions in step 1 at <a href=\"/VM/CMS\">How to install a CERN Virtual Machine</a>. Or, alternatively, install the corresponding <a href=\"/docs/cms-guide-docker\">Docker container</a>.</p>\n\t\t\t<p>To run the analysis, follow the instructions at <a href=\"https://github.com/cms-legacydata-analyses/HiForestProducerTool/tree/HIDiMuon2011\">https://github.com/cms-legacydata-analyses/HiForestProducerTool/tree/HIDiMuon2011</a>.</p> "
+    },
+    "use_with": {
+      "description": "Use this with",
+      "links": [
+        {
+          "description": "CMS 2011 heavy-ion data and releated proton-proton datasets",
+          "url": "http://opendata-dev.cern.ch/search?q=&subtype=Collision&type=Dataset&year=2011&experiment=CMS&keywords=heavy-ion%20physics"
```

Comment:
It is best not to hard-code server, i.e. use relative rather than absolute links, so that the link would work everywhere, on DEV, QA, PROD, and locally. I'll amend this while merging...

Reply:
Very good, thanks! I tried but made some mistakes and did not manage to make it working